### DISCUSSION TITLE
How to uninstall installed cs2 on linux server

### DISCUSSION THREAD

**Discussion by haozhi-ly** [kus/cs2-modded-server#100] [Q&A] [answered]
i have installed cs2 server,but i want to reinstall,i can't find unstall.sh, so i only try to delete it manually, but
What files need to be deleted?

**Comment by kus** [comment, marked as answer]
If you used the install script from this repo, you can delete the `/home/steam/` folder and that will delete mostly everything.

Then when you run the [install script](https://github.com/kus/cs2-modded-server?tab=readme-ov-file#running-on-linux) again, it will overwrite with the latest scripts.
